ui-kit/src/main/kotlin/org/p2p/uikit/utils/recycler/decoration/EightPxOffsetDecoration.kt

```diff
@@ -0,0 +1,33 @@
+package org.p2p.uikit.utils.recycler.decoration
+
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.ItemDecoration
+import android.graphics.Rect
+import android.view.View
+import kotlin.reflect.KClass
+import org.p2p.uikit.components.finance_block.FinanceBlockCellModel
+import org.p2p.uikit.model.AnyCellItem
+import org.p2p.uikit.utils.recycler.getItems
+import org.p2p.uikit.utils.toPx
+
+fun eightOffsetFinanceBlockDecoration() = EightPxOffsetDecoration(FinanceBlockCellModel::class)
+
+class EightPxOffsetDecoration(
+    private val itemCellType: KClass<out AnyCellItem>
+) : ItemDecoration() {
+
+    private val offset = 8.toPx()
```

Comment:
maybe we need to make an ability to custom offset
pass it to constructor like 

class EightPxOffsetDecoration(
    private val itemCellType: KClass<out AnyCellItem>,
    private val offsetDp = 8.toPx()
) : ItemDecoration() {